harvard-review-of-psychiatry

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard Review of Psychiatry</title>
+    <title-short>Harvard Rev Psy</title-short>
+    <id>http://www.zotero.org/styles/harvard-review-of-psychiatry</id>
+    <link href="http://www.zotero.org/styles/harvard-review-of-psychiatry" rel="self"/>
+    <author>
+      <name>Bart van den Brink</name>
+      <email>b.vandenbrink=at=kicg.nl</email>
+    </author>
+    <contributor/>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <category field="generic-base"/>
+    <issn>1465-7309</issn>
+    <updated>2024-05-01T01:48:47+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link href="https://journals.lww.com/hrpjournal/Documents/HRP%20Instructions%20for%20Authors%204.22.24.pdf" rel="journal-instructions"/>
+    <link href="https://journals.lww.com/hrpjournal/pages/default.aspx" rel="journal-site"/>
+    <link href="http://www.zotero.org/styles/nature" rel="adapted-from-this-style"/>
+  </info>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" font-style="italic" text-case="title"/>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter-precedes-last="never" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <label form="short" prefix=", "/>
+      <et-al font-style="normal"/>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="volume" type="article dataset software" match="any"/>
+      <else-if is-numeric="DOI">
+        <text variable="DOI" prefix="doi:"/>
```

Comment:
I would recommend using the modern "https://doi.org/" prefix consistently for all item types. It is strongly preferred by CrossRef and readers